docs/modules/getting_started/examples/code/getting-started.sh.j2

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# The getting started guide script
+# It uses tagged regions which are included in the documentation
+# https://docs.asciidoctor.org/asciidoc/latest/directives/include-tagged-regions/
+#
+# There are two variants to go through the guide - using stackablectl or helm
+# The script takes either 'stackablectl' or 'helm' as an argument
+#
+# The script can be run as a test as well, to make sure that the tutorial works
+# It includes some assertions throughout, and at the end especially.
+
+if [ $# -eq 0 ]
+then
+  echo "Installation method argument ('helm' or 'stackablectl') required."
+  exit 1
+fi
+
+case "$1" in
+"helm")
+echo "Adding '{{ helm.repo_name }}' Helm Chart repository"
+# tag::helm-add-repo[]
+helm repo add {{ helm.repo_name }} {{ helm.repo_url }}
+# end::helm-add-repo[]
+echo "Installing Operators with Helm"
+# tag::helm-install-operators[]
+helm install --wait commons-operator {{ helm.repo_name }}/commons-operator --version {{ versions.commons }}
+helm install --wait secret-operator {{ helm.repo_name }}/secret-operator --version {{ versions.secret }}
+helm install --wait zookeeper-operator {{ helm.repo_name }}/zookeeper-operator --version {{ versions.zookeeper }}
+helm install --wait nifi-operator {{ helm.repo_name }}/nifi-operator --version {{ versions.nifi }}
+# end::helm-install-operators[]
+;;
+"stackablectl")
+echo "installing Operators with stackablectl"
+# tag::stackablectl-install-operators[]
+stackablectl operator install \
+  commons={{ versions.commons }} \
+  secret={{ versions.secret }} \
+  zookeeper={{ versions.zookeeper }} \
+  nifi={{ versions.nifi }}
+# end::stackablectl-install-operators[]
+;;
+*)
+echo "Need to provide 'helm' or 'stackablectl' as an argument for which installation method to use!"
+exit 1
+;;
+esac
+
+echo "Installing ZooKeeper from zookeeper.yaml"
+# tag::install-zookeeper[]
+kubectl apply -f zookeeper.yaml
+# end::install-zookeeper[]
+
+echo "Installing ZNode from nifi-znode.yaml"
+# tag::install-znode[]
+kubectl apply -f nifi-znode.yaml
+# end::install-znode[]
+
+sleep 5
+
+echo "Awaiting ZooKeeper rollout finish"
+# tag::watch-zookeeper-rollout[]
+kubectl rollout status --watch statefulset/simple-zk-server-default
+# end::watch-zookeeper-rollout[]
+
+echo "Install the NiFi admin credentials from nifi-admin-credentials.yaml"
+# tag::install-nifi-credentials[]
+kubectl apply -f nifi-admin-credentials.yaml
+# end::install-nifi-credentials[]
+
+echo "Install NiFiCluster from nifi.yaml"
+# tag::install-nifi[]
+kubectl apply -f nifi.yaml
+# end::install-nifi[]
+
+sleep 5
+
+echo "Awaiting NiFi rollout finish"
+# tag::wait-nifi-rollout[]
+kubectl wait -l statefulset.kubernetes.io/pod-name=simple-nifi-node-default-0 \
+--for=condition=ready pod --timeout=-5s && \
+kubectl wait -l statefulset.kubernetes.io/pod-name=simple-nifi-node-default-1 \
+--for=condition=ready pod --timeout=-5s
```

Comment:
The timeout should probably be 5 seconds instead of -5 seconds.

```suggestion
kubectl wait -l statefulset.kubernetes.io/pod-name=simple-nifi-node-default-0 \
--for=condition=ready pod --timeout=5s && \
kubectl wait -l statefulset.kubernetes.io/pod-name=simple-nifi-node-default-1 \
--for=condition=ready pod --timeout=5s
```

Reply:
That means the check waits 5 seconds and fails. Negative means wait for a week (which is too long). I change it to 600s, otherwise nifi (with downloading images) wont get ready in time.